build.gradle

```diff
@@ -232,3 +231,36 @@ task ktlintFormat(type: JavaExec, group: "formatting") {
 compileKotlin { kotlinOptions.freeCompilerArgs = ['-Xjsr305=strict'] }
 
 apply from: 'build-tools/pkgbuild.gradle'
+
```

Comment:
Possible to start moving some of these out into separate smaller files and apply them in here?

Reply:
This part of testing code will only exist for the version that we change the way of saving metadata, so for 7.9 and 7.10. In the future, like 7.11 or 8.0, we should remove this